README.md

```diff
@@ -1,43 +1,42 @@
+It’s perfect for developers, students, or maintainers who want a clear summary of their GitHub projects.
 
-# Installation
+---
 
-- `npm install` to install all dependencies
-- create `.env` file and initialize `GH_TOKEN` or `GITHUB_TOKEN` (in order of precedence) with your Github token
+## 🛠️ How It Works
 
-# Usage (for public)
+1. **You run the tool in your terminal.**
+2. **Repo-Report connects to GitHub using your personal access token (PAT).**  
+   This is a password-like key GitHub gives you to allow tools to access your repos.
+3. **It scans every repository you have access to** — personal or organizational.
+4. **You get a report in your terminal** that shows useful details about each repo.
 
-- generate a personal access token using github [here](https://github.com/settings/tokens) and add the `repo` scope to it.
-- on the terminal run `export GH_TOKEN=<the personal access token generated>`
-- run `npx repo-report`
+Here’s what it looks at by default:
+- ✅ Are GitHub Issues turned on?
+- 🗂️ Are GitHub Projects enabled?
+- 📜 Does the repo have a license file?
+- 📖 Is the GitHub Wiki enabled?
+- 🔐 Is the repo public, private, or internal?
+- 🧪 Are GitHub Actions workflows present?
+- 🧩 Are topics/tags set?
+- 🕒 When was the last push?
+- 🧑‍🤝‍🧑 How many collaborators are there?
 
-# Usage (for Contributors)
+---
 
-- execute `./bin/run` to get a report of all your repositories in the terminal
+## 🚀 Getting Started
 
-[package-url]: https://npmjs.org/package/repo-report
-[npm-version-svg]: https://versionbadg.es/ljharb/repo-report.svg
-[deps-svg]: https://david-dm.org/ljharb/repo-report.svg
-[deps-url]: https://david-dm.org/ljharb/repo-report
-[dev-deps-svg]: https://david-dm.org/ljharb/repo-report/dev-status.svg
-[dev-deps-url]: https://david-dm.org/ljharb/repo-report#info=devDependencies
-[npm-badge-png]: https://nodei.co/npm/repo-report.png?downloads=true&stars=true
-[license-image]: https://img.shields.io/npm/l/repo-report.svg
-[license-url]: LICENSE
-[downloads-image]: https://img.shields.io/npm/dm/repo-report.svg
-[downloads-url]: https://npm-stat.com/charts.html?package=repo-report
-[codecov-image]: https://codecov.io/gh/ljharb/repo-report/branch/main/graphs/badge.svg
-[codecov-url]: https://app.codecov.io/gh/ljharb/repo-report/
-[actions-image]: https://img.shields.io/endpoint?url=https://github-actions-badge-u3jn4tfpocch.runkit.sh/ljharb/repo-report
-[actions-url]: https://github.com/ljharb/repo-report/actions
+Installation
+npm install to install all dependencies
+create .env file and initialize GH_TOKEN or GITHUB_TOKEN (in order of precedence) with your Github token
+Usage (for public)
+generate a personal access token using github here and add the repo scope to it.
+on the terminal run export GH_TOKEN=<the personal access token generated>
+run npx repo-report
+Usage (for Contributors)
+execute ./bin/run to get a report of all your repositories in the terminal
```

Comment:
Please note how this last paragraph looks.

Can we make sure it's more readable?

<img width="2248" height="1337" alt="image" src="https://github.com/user-attachments/assets/9e64c98f-20de-4f78-8119-067e0d6dad43" />